### PR TITLE
feat(task2):created layout

### PR DESCRIPTION
**Pull request by Emigrate**

### CODE REVIEW

File: task2/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+<main>
+    <header>HEADER</header>
+    <div class="content">
+        <aside>ASIDE</aside>
+        <article>
+            <div class="textArticle">
```
Comment (Sacronux) [inline task2/index.html:16]: css naming convention is not camel case

File: task2/index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                ARTICLE
+            </div>
+            <div class="imgcontainer">
```
Comment (Sacronux) [inline task2/index.html:19]: word must be splitten by '-'

File: task2/style.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
+main {
+    font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+    font-size: 50px;
+    color: rgb(52, 141, 214);
+}
+
+header {
+    width: 99%;
```
Comment (Sacronux) [inline task2/style.css:8]: why 99%?

File: task2/style.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    height: 200px;
+    background-color: paleturquoise;
+    margin: 10px 10px 10px 10px;
```
Comment (Sacronux) [inline task2/style.css:11]: use `margin: 10px`

File: task2/style.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+article {
+    background-color: orange;
+    width: 100%;
+    height: 100%;
+    margin: 10px 10px 10px 10px;
+    border-radius: 10px;
+}
+
+footer {
+    width: 99%;
```
Comment (Sacronux) [inline task2/style.css:44]: 100%

File: task2/style.css [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    height: 200px;
+    background-color: paleturquoise;
+    border-radius: 20px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 10px;
+    margin-top: 20px;
```
Comment (Sacronux) [inline task2/style.css:52]: duplication of styles, use `margin: 20 10px 10px`

**Review by crucas — CHANGES_REQUESTED** [review]

**Review by crucas — APPROVED** [review]

**Pull request merged**